scripts/contributors.sh

```diff
@@ -29,20 +29,22 @@
 # RELEASE-NOTES.
 #
 
-start=$1
+set -eu
+
+start="${1:-}"
 
 if test "$start" = "-h"; then
-    echo "Usage: $0 <since this tag/hash> [--releasenotes]"
-    exit
+  echo "Usage: $0 <since this tag/hash> [--releasenotes]"
+  exit
 fi
 if test -z "$start"; then
-    start=`git tag --sort=taggerdate | grep "^curl-" | tail -1`;
-    echo "Since $start:"
+  start=$(git tag --sort=taggerdate | grep "^curl-" | tail -1)
+  echo "Since $start:"
 fi
 
 # We also include curl-www if possible. Override by setting CURLWWW
-if [ -z "$CURLWWW" ] ; then
-    CURLWWW=../curl-www
+if [ -z "${CURLWWW:-}" ] ; then
+  CURLWWW=../curl-www
 fi
```

Comment:
Maybe the cooler way to write this in proper shell is this?

```suggestion
CURLWWW="${CURLWWW:-../curl-www}"
```